upload-current.sh

```diff
@@ -1,77 +1,89 @@
 #!/usr/bin/env bash
+#
+# Upload trained model s3 in a format compatible with DeepRacer model import functionality.
+# Example usage to upload the best model:
+#     ./upload-current.sh aws-deepracer-XXX model1 -b
+#
 
 S3_BUCKET=$1
 S3_PREFIX=$2
 
 MODEL_DIR=data/minio/bucket/current/model/
 
-while getopts ":c:" opt; do
+echo "Uploading to model ==>  s3://$S3_BUCKET/$S3_PREFIX  <=="
+
+USE_BEST=false
+while getopts ":c:b" opt; do
 case $opt in
-c) CHECKPOINT="$OPTARG"
-;;
-\?) echo "Invalid option -$OPTARG" >&2
-;;
+    c) CHECKPOINT="$OPTARG"
+       ;;
+    b) USE_BEST=true
+       ;;
+    \?) echo "Invalid option -$OPTARG" >&2
+        ;;
 esac
 done
 
 CHECKPOINT_FILE=$MODEL_DIR"deepracer_checkpoints.json"
 if [ ! -f ${CHECKPOINT_FILE} ]; then
   echo "Checkpoint file not found!"
   exit 1
-else
-  echo "found checkpoint index file "$CHECKPOINT_FILE
-fi;
+fi
+echo "found checkpoint index file "$CHECKPOINT_FILE
 
 if [ -z "$CHECKPOINT" ]; then
-  echo "Checkpoint not supplied, checking for latest checkpoint"
-
-  LAST_CHECKPOINT=`cat $CHECKPOINT_FILE |jq ".last_checkpoint.name"`
-  BEST_CHECKPOINT=`cat $CHECKPOINT_FILE |jq ".best_checkpoint.name"`
-  
-  CHECKPOINT=$LAST_CHECKPOINT
-
-  echo "latest checkpoint = "$CHECKPOINT
+  #echo "Checkpoint not supplied, checking for latest checkpoint"
+  LAST_CHECKPOINT=`cat $CHECKPOINT_FILE |jq ".last_checkpoint.name" | sed s/\"//g`
```

Comment:
You shouldn't need LAST_CHECKPOINT, BEST_CHECKPOINT or even just CHECKPOINT would be enough.

Reply:
I always use the command line option -b, so end up using BEST_CHECKPOINT.
But the script in Matt's repo was defaulting to last checkpoint, so I guess that others prefer to upload last checkpoint rather than best checkpoint.

Reply:
As discussed on slack, we'll default to last checkpoint. Best checkpoint can be used with the cmd line option -b.